test/protractor.conf.js

```diff
@@ -11,6 +11,7 @@ module.exports.config = {
   specs: ["*.spec.js"],
   onPrepare() {
     browser.waitForAngularEnabled(false);
+    // afterEach(() => browser.sleep(1000));
```

Comment:
We're using 'headless', this isn't useful for us, Why keep?

Reply:
When debugging I comment the headless config and this turns to be useful.